From d532905072955a1a0b6f7ad19b7c11e2df441e46 Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Thu, 11 May 2017 16:00:39 +0300
Subject: [PATCH 1792/5242] MLK-14800: fbdev: Refactor printed information

commit  c108d3e3b1a45156139c95aa83b3db106daab7a3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Debug info should not be printed using pr_info on init. Use pr_debug
instead.
Also, use the driver name, since there is more than one MIPI DSI driver.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mipi_dsi_samsung.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/mipi_dsi_samsung.c b/drivers/video/fbdev/mxc/mipi_dsi_samsung.c
index 491ea68..3b2a0ee 100644
--- a/drivers/video/fbdev/mxc/mipi_dsi_samsung.c
+++ b/drivers/video/fbdev/mxc/mipi_dsi_samsung.c
@@ -925,7 +925,7 @@ static int __init mipi_dsi_init(void)
 		return err;
 	}
 
-	pr_info("MIPI DSI driver module loaded\n");
+	pr_debug("MIPI DSI driver module loaded: %s\n", mipi_dsi_driver.driver.name);
 
 	return 0;
 }
-- 
1.7.9.5

